### ISSUE TITLE
Edge Browser has been updated to Chromium based - App no longer functions

### ISSUE THREAD

**Issue by tcreek** [janch32/Spotify-WPW#20]
What can we do about this?

**Comment by janchaloupka** [comment]
Hello, this app is no longer being actively developed and it targeted Windows 10 Mobile only. Do you have a source for your claim, that MS pushed update to W10M? Because I cannot find anything about chromium based edge for mobile windows.

But if you want, feel free to fork this repo and you can continue working on this app.

**Comment by tcreek** [comment]
You should reactivate it.

Here is the update notes:

https://support.microsoft.com/en-us/help/4522812/windows-10-update-kb4522812

However, it does not mention the Browser change. On the Feedback hub, that was a massive request for such an update for the last update.

And with that, I present you this screenshot.

![wp_ss_20191211_0001 1](https://user-images.githubusercontent.com/4149717/70661362-b4285380-1c29-11ea-961c-fe3bb80ba949.png)

**Comment by janchaloupka** [comment]
User agent does not reflect actual browser rendering engine. In fact i just turned on my old Lumia and the user agent is exactly the same as is on your screenshot.

I suspect that the Spotify team simply pushed update to the web player that broke it's function on mobile ms edge.

**Comment by tcreek** [comment]
Okay, Thanks for the verification of browser.

I went ahead and forked. 

Here is a screenshot of what the app is currently displaying.

Maybe you can give some hints & tips on proceeding

![wp_ss_20191211_0002 1](https://user-images.githubusercontent.com/4149717/70662466-e044d400-1c2b-11ea-82f2-4cd3b2318535.png)

Thanks

**Comment by asp3n-hayley** [comment]
> 
> 
> Okay, Thanks for the verification of browser.
> 
> I went ahead and forked.
> 
> Here is a screenshot of what the app is currently displaying.
> 
> Maybe you can give some hints & tips on proceeding
> 
> ![wp_ss_20191211_0002 1](https://user-images.githubusercontent.com/4149717/70662466-e044d400-1c2b-11ea-82f2-4cd3b2318535.png)
> 
> Thanks

I'm not an expert on mobile apps, but I have a nokia lumia 630 that I modified to run win10M even though it only supports 8.1, and I could have sworn last time I used it(~6 months ago) that microsoft had added an official spotify app in the windows store, thus rendering this web player obsolete. Am I mistaken, or not following this issue?

**Comment by tcreek** [comment]
Microsoft did not add any such software. It was done by Spotify. They put both a desktop, and mobile version in the Windows store. The mobile version was actually done for Windows 8.1, but still runs on Windows 10 (it does for me sometimes.) I have an old Windows 8.1 phone which the Spotify player still functions. The search function was disabled for some unknown reason.

**Comment by asp3n-hayley** [comment]
> Microsoft did not add any such software. It was done by Spotify. They put both a desktop, and mobile version in the Windows store. The mobile version was actually done for Windows 8.1, but still runs on Windows 10 (it does for me sometimes.) I have an old Windows 8.1 phone which the Spotify player still functions. The search function was disabled for some unknown reason.

You're right, that's what I meant. My point was, if spotify is available in the store, why try to create a functional web player, with as many updates as edge gets?

**Comment by janchaloupka** [comment]
I created this app because because official app for wmobile has been abandoned and lacks some new features that I really wanted. Stuff like daily mixes, Spotify connect and podcasts. Obviously due to changes the Spotify team made to the web app it no longer works on windows mobile, so the official app is the only way to play songs on Spotify.

As for what the issue is I honestly have no idea. Even back then the web player did not work in the browser. It took me about a week to find reliable way how to fool the webpage to thinking that it is running on desktop. 

But I suspect that the issue is with new DRM or new feature that is not supported by the outdated mobile edge browser, but that is just my speculation.

**Comment by asp3n-hayley** [comment]
> 
> 
> I created this app because because official app for wmobile has been abandoned and lacks some new features that I really wanted. Stuff like daily mixes, Spotify connect and podcasts. Obviously due to changes the Spotify team made to the web app it no longer works on windows mobile, so the official app is the only way to play songs on Spotify.
> 
> As for what the issue is I honestly have no idea. Even back then the web player did not work in the browser. It took me about a week to find reliable way how to fool the webpage to thinking that it is running on desktop.
> 
> But I suspect that the issue is with new DRM or new feature that is not supported by the outdated mobile edge browser, but that is just my speculation.

That makes perfect sense. I briefly used a windows phone, and during that time I found it greatly wanting generally speaking, but the spotify app was sorely lacking. 
Nonetheless, I appreciate a skilled dev catering to the lesser known devices and not just the flagships. Good luck!
